pkg/keysv2/evm_tx.go

```diff
@@ -0,0 +1,154 @@
+// Package keysv2 provides key management functionality for EVM transactions and OCR2 keyrings.
+package keysv2
```

Comment:
Typically a v2 package is under a `<original>/v2` directory, but with the same name `package <original>`, so that importers reference it with the `/v2` suffix, but the import name doesn't change.
```suggestion
package keys
```
```diff
-import common/pkg/keys
+import common/pkg/keys/v2

keys.Thing
```